Gopkg.toml

```diff
@@ -32,6 +32,10 @@ required = [
   "github.com/vektra/mockery/cmd/mockery"
 ]
 
+[[override]]
+  version = "v1.1.0"
+  name = "github.com/pseudomuto/protoc-gen-doc"
```

Comment:
Why this protoc-gen-doc is included in this PR? Are we actually using this? They already have docker image and posted in the readme how to use this and thier using this like we do with our docker scripts.

Reply:
We already require this pkg in the Gopkg file, but I just added these lines to lock its version. Otherwise _dep ensure_ may install an older version which causes to this issue: https://github.com/mesg-foundation/core/pull/541#pullrequestreview-175688242

Reply:
Ok